.travis.yml

```diff
@@ -19,6 +19,8 @@ script:
 - yarn test:helpers
 - yarn test:typescript
 - yarn test:license
+- ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --reporter lcovonly -- -R spec
```

Comment:
You can move this to a scripts command in package json just to maintain uniformity..

Reply:
Just a suggestion